### ISSUE TITLE
1.16.1 Crashes on Startup

### ISSUE THREAD

**Issue by P4YN-Ki** [Ladysnake/Cardinal-Components-API#53]
Cardinal-Components-API v2.4.2 will crash if the game is started with Optifabric also enabled. Disabling either 1 or the other will enable the game to boot correctly, but not together. Issue repeatable with versions 2.4.0, 2.4.1 & 2.4.2

**Comment by Pyrofab** [comment]
Huh, 2.4.2 did not fix it ? Could you post a crash report ?

**Comment by P4YN-Ki** [comment]
That from the Logs Folder? I can certainly drop you that. There are some other issues in there, I'm aware. Which I've fixed in another instance, but this one I can't seem to correct.
I should note I actually got it working with the version meant for 1.16.2? maybe 3 on a 1.16.1 instance, but upon activating Shaders of any kind I get the loading screen over the top of the gameplay and can still hear the sounds ingame.

[latest.log](https://github.com/OnyxStudios/Cardinal-Components-API/files/5211900/latest.log)
